validator/client/src/test/java/tech/pegasys/teku/validator/client/signer/SignerTest.java

```diff
@@ -53,11 +53,11 @@ public void shouldCreateRandaoReveal() {
   }
 
   @Test
-  public void shouldSignBlock() {
+  public void shouldSignBlock1() {
     final BeaconBlock block = dataStructureUtil.randomBeaconBlock(10);
     final BLSSignature signature = dataStructureUtil.randomSignature();
     final Bytes expectedSigningRoot =
-        Bytes.fromHexString("0xc0e4ed8375c98504b262f610f217d31ebf109f0f73c164362090c6ad7d4770c1");
+        Bytes.fromHexString("0xfa8b3cfed0268ed15e354e84db5558eb76ad30737a86d6d057615e331ff30d44");
```

Comment:
Ugh. Why did this change? Is `dataStructureUtil.randomSignature();` using a different seed value? That whole thing looks a bit fragile to me. (Not your problem!)

Reply:
All the BLS random items are generated using random secret key. The random key was generated with Mikuli `RAND` and blst uses key material to derive secret key from. It doesn't matter for normal use, but for tests the secret key generated from the same seed differed between Mikuli and Blst. So I did implementation independent secret key generation from seed for testing purposes and update this hash to actual value. 

Yep it looks fragile but seems to work fine: the same seed should always yields the same random structure. With the exception may be of this change  :)